module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp

```diff
@@ -145,7 +147,7 @@ namespace module::behaviour::planning {
             }
         });
 
-        on<Every<20, Per<std::chrono::seconds>>,
+        on<Every<40, Per<std::chrono::seconds>>,
```

Comment:
Maybe a comment with why this should be equal to or greater than soccer strategy.